node/libs/crypto/src/bn254/tests.rs

```diff
@@ -1,8 +1,6 @@
-use std::iter::repeat_with;
-
-use rand::{rngs::StdRng, Rng, SeedableRng};
-
 use crate::bn254::{AggregateSignature, PublicKey, SecretKey, Signature};
+use rand::{rngs::StdRng, Rng, SeedableRng};
+use std::iter::repeat_with;
```

Comment:
Off-topic: Weird that this incorrect formatting isn't caught in CI.

Reply:
you mean multiple import blocks?